media/videostitcher/create_cdn_key_test.go

```diff
@@ -54,9 +50,8 @@ func TestCreateMediaCDNKey(t *testing.T) {
 			r.Errorf("createCDNKey got: %v Want to contain: %v", got, mediaCDNKeyName)
 		}
 	})
-	t.Cleanup(func() {
-		teardownTestCreateCDNKey(mediaCDNKeyName, t)
-	})
+
+	deleteTestCDNKey(mediaCDNKeyName, t)
```

Comment:
Here and elsewhere, you can still call the function from within a `t.Cleanup` to ensure that it runs even if the test errors out.

Reply:
Done